.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+@jesper7 @mjasion @omesser
```

Comment:
@omesser do you think we should add you Codeowners?  You will be asked for every approve ;-)

Reply:
sure, no problem